handbook/engineering/search/iteration_log.md

```diff
@@ -14,9 +14,41 @@ This document contains the goals and work log for the search team's [2-week iter
 
 ## 2020-09-21 to 2020-10-02
 
-### $GOAL_OR_THEME
+### Scale indexed search to 500k repositories
 
-- **Owner(s):** $OWNER
+- **Owner(s):** Stefan
+- **Outcomes:**
+    - $DESIRED_OUTCOME
+- **Work log:**
+    - YYYY-MM-DD: $UPDATE
+
+### Streaming search
+
+- **Owner(s):** Keegan
+- **Outcomes:**
+    - $DESIRED_OUTCOME
+- **Work log:**
+    - YYYY-MM-DD: $UPDATE
+
+### Enterprise homepage
+
+- **Owner(s):** Juliana
+- **Outcomes:**
+    - $DESIRED_OUTCOME
+- **Work log:**
+    - YYYY-MM-DD: $UPDATE
+
+### Search tour
+
+- **Owner(s):** Farhan
+- **Outcomes:**
+    - $DESIRED_OUTCOME
+- **Work log:**
+    - YYYY-MM-DD: $UPDATE
+
+### Search expressions
+
+- **Owner(s):** Rijnard
 - **Outcomes:**
     - $DESIRED_OUTCOME
```

Comment:
@rvantonder can you fill in your plan for the next iteration? If "search expressions" doesn't match what you were planning to focus on feel free to change it.